App.js

```diff
@@ -25,6 +25,7 @@ import {
 } from 'react-native/Libraries/NewAppScreen';
 
 const App: () => React$Node = () => {
+  console.log('Github');
```

Comment:
> __[Goodcheck]__ GitHub is GitHub, not Github ([view](https://sider.review/gh/repos/269516938/pulls/2#issue-4034486))
>
> <details>
>
> You may be misspelling the name of the service!
>
> | Rule |
> | ---- |
> | `com.example.github` |
>
> You can [close this issue](https://sider.review/gh/repos/269516938/pulls/2#issue-4034486) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue, because:
- When you mean a service different from GitHub